go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	code.cloudfoundry.org/clock v1.1.0
 	code.cloudfoundry.org/credhub-cli v0.0.0-20230410130651-444cd52cc23d
 	code.cloudfoundry.org/garden v0.0.0-20230418162742-6c127706d54f
+	code.cloudfoundry.org/lager v2.0.0+incompatible
```

Comment:
Can you run `go mod tidy` as well. I think this is left over from when you import `code.cloudfoundry.org/lager`